content/10.diagnostics.md

```diff
@@ -309,16 +317,16 @@ Tests that identify SARS-CoV-2 using nucleic-acid-based technologies will identi
 Among molecular tests, different technologies have different sensitivities and specificities.
 In general, specificity is high, and even then, the public health repercussions of a false positive can generally be mitigated with follow-up testing.
 On the other hand, a test's sensitivity, which indicates the risk of a false-negative response, can pose significant challenge to large-scale testing.
-False negatives are a significant concern for several reason.
+False negatives are a significant concern for several reasons.
 Importantly, clinical reports indicate that it is imperative to exercise caution when interpreting the results of molecular tests for SARS-CoV-2 because negative results do not necessarily mean a patient is virus-free [@doi:10.1128/JCM.00297-20].
 To reduce occurrence of false negatives, correct execution of the analysis is crucial [@doi:10.1038/d41587-020-00002-2].
 Additionally, PCR-based tests can remain positive for a much longer time than the virus is likely to be actively replicating [@doi:10.1056/NEJMcp2117115], raising concerns about their informativeness after the acute phase of the disease.
-Hence why the CDC has advised individuals who suspect they have been re-infected with SARS-CoV-2 to avoid using diagnostic tests within 90 days of receiving a previous positive test [@url:https://www.cdc.gov/coronavirus/2019-ncov/your-health/quarantine-isolation.html].
+Hence the CDC has advised individuals who suspect they have been re-infected with SARS-CoV-2 to avoid using diagnostic tests within 90 days of receiving a previous positive test [@url:https://www.cdc.gov/coronavirus/2019-ncov/your-health/quarantine-isolation.html].
 
 Additionally, the emerging nature of the COVID-19 pandemic has introduced some challenges related to uncertainty surrounding interactions between SARS-CoV-2 and its human hosts.
 For example, viral shedding kinetics are still not well understood but are expected to introduce a significant effect of timing of sample collection on test results [@doi:10.1038/d41587-020-00002-2].
 Similarly, the type of specimen could also influence outcomes, as success in viral detection varies among clinical sample types [@doi:10.1038/d41587-020-00002-2; @doi:10/gjrfns; @doi:10.2807/1560-7917.ES.2020.25.50.2000568].
-With CRISPR-based testing strategies, the gRNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur.
+With CRISPR-based testing strategies, the gRNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur.<!--Clarification: I don’t think this should be “patient’s genome” because the virus doesn’t integrate into human genomes?-->
```

Comment:
@SystemsResearch @nilswellhausen would either of you be able to confirm whether this line is what was intended?

Reply:
@jessegmeyerlab I found a reference that I think agrees with your concern. I propose this change:

```suggestion
With CRISPR-based testing strategies, the gRNA can recognize off-target interspersed sequences in the viral genome [@doi:10.3389/fcimb.2020.576875], potentially resulting in false positives and a loss of specificity.
```

Reply:
That makes more sense thank you!